crescent/client.py

```diff
@@ -220,14 +226,23 @@ async def ping(ctx: crescent.Context):
             await ctx.respong("Pong")
         ```
         """
-        if command is None:
+        if obj is None:
             return self.include
 
-        add_hooks(self, command)
+        if isinstance(obj.metadata, AppCommandMeta):
+            obj.metadata.add_hooks(self.command_hooks, after=False)
+            obj.metadata.add_hooks(self.command_after_hooks, after=True)
+        if isinstance(obj.metadata, EventMeta):
+            obj.metadata.add_hooks(  # pyright: ignore [reportUnknownMemberType]
```

Comment:
why is the type unknown here, but known for AppCommandMeta?

Reply:
Pyright doesn't know the type for `EventMeta` because of the genetic. I ignored the error because the type of the generic doesn't matter to the code at all.